src/status_im/contexts/wallet/common/account_switcher/view.cljs

```diff
@@ -13,20 +15,24 @@
     nil))
 
 (defn view
-  [{:keys [on-press accessibility-label icon-name switcher-type margin-top]
+  [{:keys [type on-press accessibility-label icon-name switcher-type margin-top]
     :or   {icon-name           :i/close
            accessibility-label :top-bar
            switcher-type       :account-options}}]
   (let [{:keys [color emoji watch-only?]} (rf/sub [:wallet/current-viewing-account])
         networks                          (rf/sub [:wallet/network-details])]
     [quo/page-nav
-     {:icon-name           icon-name
+     {:type                (or type :no-title)
```

Comment:
Had to keep it this way as we use this common navigation view in most of the wallet screens.
Only the account main screen displays the network icons (dropdown)